ram_msgs/AdditiveManufacturingPose[] selected_poses
bool temporary
---
